templates/login.html

```diff
@@ -49,5 +49,12 @@ <h1 class="text-2xl font-bold tracking-tight text-center">
       ></span>
       Sign in
     </button>
+    <button
```

Comment:
We should support deployments without any SSO functionality. If the google oauth provider is not configured, this button should be hidden.